cypress/specs/docs/components.spec.ts

```diff
@@ -184,4 +184,31 @@ describe('Docs Site: Components', () => {
       })
     })
   })
+
+  describe('when browsing directly to a section', () => {
+    const itemIndex = 3
+
+    before(() => {
+      // Block newrelic.js due to issues with Cypress networking
```

Comment:
It would be nice to move this somewhere so we don't need to keep duplicating it.